services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -229,9 +254,24 @@ public TopicManagementService(Map<String, Object> props, String serviceName) {
     _topic = config.getString(CommonServiceConfig.TOPIC_CONFIG);
     _partitionsToBrokerRatio = config.getDouble(CommonServiceConfig.PARTITIONS_TO_BROKER_RATO_CONFIG);
     _scheduleIntervalMs = config.getInt(TopicManagementServiceConfig.REBALANCE_INTERVAL_MS_CONFIG);
-    String zkUrl = config.getString(CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG);
-    _zkUtils = ZkUtils.apply(zkUrl, ZK_SESSION_TIMEOUT_MS, ZK_CONNECTION_TIMEOUT_MS, JaasUtils.isZkSecurityEnabled());
+    _zkConnect = config.getString(CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG);
```

Comment:
Why do you change this from local variable to class variable? It is not used anywhere else in the class.